src/instantsearch/widget.js

```diff
@@ -31,19 +35,15 @@ class AlgoliaPlacesWidget {
         },
       } = opts;
 
-      this.state.position = `${lat},${lng}`;
-      this.state.query = value;
-
+      this.query = value;
       helper
         .setQueryParameter('insideBoundingBox')
-        .setQueryParameter('aroundLatLng', this.state.position)
+        .setQueryParameter('aroundLatLng', `${lat},${lng}`)
```

Comment:
`setQueryParameter` will set the page to 0 therefore I'm not sure the bug is going to fixed here.

Reply:
The highlighting on GH makes it hard to read, but this bit of code is in the `change` event code block. It is not executed during the `init` step but only once the user selects a value from the places dropdown.

Reply:
Oh ok my bad.